src/utils.ts

```diff
@@ -20,3 +20,13 @@ export function toBaseUnits<T extends Currency>(
 ): BigNumber {
     return BigNumber.from(amount.quotient.toString());
 }
+
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+export function invariant(condition: any, message: string): asserts condition {
```

Comment:
will have a type for `condition` in the future I'm assuming

Reply:
Nah, it's meant to accept anything and check for it being truthy/falsey so the `any` type is here to stay alas.